data/tlg0059/tlg037/tlg0059.tlg037.1st1K-grc1.xml

```diff
@@ -50,25 +51,21 @@
 <pubPlace>United States</pubPlace>
 </publicationStmt>
 <sourceDesc>
-<listBibl xml:lang="lat">
+<listBibl>
```

Comment:
`<listBibl>` is not standard inclusion?

Reply:
hi @lcerrato you are right, I think I had just gotten out of habit of eliminating it. Thanks!